contracts/ProtocolReserve/ProtocolShareReserve.sol

```diff
@@ -322,11 +323,11 @@ contract ProtocolShareReserve is
         address asset,
         IncomeType incomeType
     ) public override(IProtocolShareReserve) nonReentrant {
-        if (!ComptrollerInterface(comptroller).isComptroller()) revert InvalidAddress();
+        if (!IComptroller(comptroller).isComptroller()) revert InvalidAddress();
         if (asset == address(0)) revert InvalidAddress();
```

Comment:
We can use `ensureNonZeroAddress` validator from `soldiity-utilities` packages

Reply:
Done https://github.com/VenusProtocol/protocol-reserve/pull/9/commits/605f5b6802449649d52bf53db2f9a038590a9119